config/crd/bases/atlas.mongodb.com_atlasdeployments.yaml

```diff
@@ -255,10 +255,6 @@ spec:
                       object, you must also specify the providerSettings.autoScaling
                       object.
                     properties:
-                      autoIndexingEnabled:
```

Comment:
Not sure if we have to remove this field as this introduces a breaking change to the CRD. Maybe we should just mark this one as deprecated and not used it, until the v2.x when it can be completely removed?